deployment/cfn/worker.py

```diff
@@ -240,6 +248,10 @@ def get_cloud_config(self):
         return ['#cloud-config\n',
                 '\n',
                 'write_files:\n',
+                '  - path: /etc/rf.d/env/RF_STACK_COLOR\n',
```

Comment:
Oh, I read this backwards. This is the cloud-config that creates the `RF_STACK_COLOR` env dir on the app/worker instances, but does not actually get used anywhere (yet). Disregard!